operations/companies.md

```diff
@@ -50,46 +50,56 @@ Returns all company profiles of the enterprise, possibly filtered by identifiers
 
 ### Response
 
-```javascript
+```json
```

Comment:
Although this is correct and nicer, it creates inconsistency between this and other endpoints examples. It should be changed repository-wide in some other PR. cc @MikeAdamsMews

Reply:
I'll revert this change for now

Reply:
Yes, I agree with @vitezp, I'll pick it up separately. Interestingly, both json and javascript versions render exactly the same in GitBook, even though they are different in GitHub.

Reply:
@MikeAdamsMews Doesn't GitBook require extra configuration for it to render json as markdown properly?